test/lib/import/colorectal/providers/manchester/manchester_handler_colorectal_test.rb

```diff
@@ -58,7 +58,7 @@ def setup
                                             Import::Helpers::Colorectal::Providers::R0a::R0aConstants::PASS_THROUGH_FIELDS_COLO)
       @handler.process_fields(record)
       testscope = @handler.testscope_from_rawfields(genocolorectal, record)
```

Comment:
Not sure we need the `testscope` variable? There are two further instances below